addons/hr_holidays/static/src/components/day_month_selection/day_month_selection.js

```diff
@@ -0,0 +1,31 @@
+import { registry } from "@web/core/registry";
+import { SelectionField, selectionField } from "@web/views/fields/selection/selection_field";
+
+export class DayMonthSelectionField extends SelectionField {
+    static props = {
+        ...SelectionField.props,
+        selectedMonth: { type: String, optional: true },
+    };
+
+
+    get options() {
+        const selectedMonth = this.props.record.data[this.props.selectedMonth];
+        let date = new Date(2024, selectedMonth, 0);
+        let days = date.getDate();
```

Comment:
```suggestion
        const date = new Date(2024, selectedMonth, 0);
        const days = date.getDate();
```

Use const instead of let where values don't change:

Reply:
done